woocommerce/woocommerce-android
# Free Trial: Adjust store creation use case

Fix issue #8781 

Summary
==========
Takes advantage of the code removal in https://github.com/woocommerce/woocommerce-android/pull/8775 and https://github.com/woocommerce/woocommerce-android/pull/8776 and simplifies how the `CreateFreeTrialStore` use case works.

How to Test
==========
1. Simply test the Free Trial store creation scenario and make sure it works as expected.

Update release notes:
